arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Log Analytics Client for LinkTargets, StorageInsightConfigs and SavedSearches.",
+    "description": ".Net client wrapper for the REST API for Azure Log Analytics",
```

Comment:
This is a swagger spec that will be used for multiple reasons. Please do no use the word client in the title and description. The spec will be used to generate clients in node.js, go , java, php, python, ruby and more languages in the future. Hence mentioning .NET also makes no sense.